### PR TITLE
Minor fixes to complexes pipeline

### PR DESCRIPTION
**Pull request by aggreen**
See commits. The following major changes were made:

1. correctly identifying paralogs for best reciprocal hit pairing. This was broken due to incorrect handling of outputs of parse_header.

2. couplings stage now outputs a CouplingScores_inter.csv file.

### CODE REVIEW

File: evcouplings/complex/similarity.py [hunk 1]
```diff
@@ -113,13 +113,14 @@ def find_paralogs(target_id, annotation_data, identity_threshold):
...
-    base_query = parse_header(target_id)
+    # output of parse_header is (ID, region_start, region_end)
+    base_query_id, _, _ = parse_header(target_id)
 
     # get all the rows that have an id that contains the
     # query id. This includes the focus sequence and its hit to
     # itself in the database.
 
-    contains_annotation = [ True if base_query in x else False for x in annotation_data.id.str ]
+    contains_annotation = [ True if base_query_id in x else False for x in annotation_data.id]
```
Comment (thomashopf) [inline evcouplings/complex/similarity.py]: Testing of boolean - this should just be 
`contains_annotation = [base_query_id in x for x in annotation_data.id]`

Comment (aggreen) [reply evcouplings/complex/similarity.py]: fixed

**Comment by aggreen** [comment]
I resolved the conflicts but there is one small decision to be made: I want to keep in three lines of code in the couplings/protocol.py that write just the inter ECs to a file. This doesn't fit nicely into the new refactored code, because it would be a different ec_filter parameter than is currently used for the rest of the computation. I'd have to call _postprocess_ecs twice. See the TODO I left in the code.

**Comment by thomashopf** [comment]
I can't see the TODO but I can see the issue - I think we have two possibilities here:

1) After the call to the postprocessing function, we load the EC file using the respective entry in the outcfg in the complex protocol, and create the second table for the inter ECs. Not the prettiest solution but the simplest.

2) We generalize the postprocessing so it takes a list of tables to generate (name of output key / filename suffix / query string) instead of just one

For now I would just do 1) and then, depending what other protocols we get in the future, refactor according to 2) eventually.

**Comment by aggreen** [comment]
Derp - I resolved the conflicts in the web editor but never committed the changes, hence no TODO statement. I went with solution #1.

**Comment by b-schubert** [comment]
Please PR on develop now

**Pull request closed without merging**